src/chunkserver/chunkserver_entry.cc

```diff
@@ -0,0 +1,1578 @@
+/*
+   Copyright 2013-2015 Skytechnology sp. z o.o.
+   Copyright 2023      Leil Storage OÜ
+
+   This file is part of SaunaFS.
+
+   SaunaFS is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, version 3.
+
+   SaunaFS is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with SaunaFS. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "common/platform.h"
+
+#include "chunkserver/chunkserver_entry.h"
+
+#include <fcntl.h>
+#include <netinet/in.h>
+#include <sys/types.h>
+#include <sys/uio.h>
+#include <syslog.h>
+#include <unistd.h>
+#include <cassert>
+#include <cerrno>
+#include <cinttypes>
+#include <cstdint>
+#include <cstdlib>
+#include <cstring>
+#include <ctime>
+#include <memory>
+#include <set>
+
+#include "chunkserver/bgjobs.h"
+#include "chunkserver/hdd_readahead.h"
+#include "chunkserver/hddspacemgr.h"
+#include "chunkserver/network_stats.h"
+#include "common/charts.h"
+#include "protocol/cltocs.h"
+#include "protocol/cstocl.h"
+#include "protocol/cstocs.h"
+#include "common/datapack.h"
+#include "common/event_loop.h"
+#include "common/saunafs_version.h"
+#include "common/massert.h"
+#include "protocol/SFSCommunication.h"
+#include "common/legacy_vector.h"
+#include "protocol/packet.h"
+#include "slogger/slogger.h"
+#include "common/sockets.h"
+#include "devtools/request_log.h"
+#include "devtools/TracePrinter.h"
+
+constexpr uint32_t kMaxPacketSize = 100000 + SFSBLOCKSIZE;
```

Comment:
Consider maybe explaining why 100 000 plus SFSBLOCKSIZE is the maximum allowed Packet size

Reply:
Not in the scope of this change.
will be handled as part of the refactoring, but this change is only moving to new files to facilitate the review.